src/lib/containers/InfiniteQueryWrapper.tsx

```diff
@@ -150,95 +164,16 @@ export function InfiniteQueryWrapper(props: InfiniteQueryWrapperProps) {
     return infiniteData?.pages[currentPage].responseBody
   }, [currentPage, infiniteData])
 
-  useDeepCompareEffect(() => {
-    if (onQueryChange) {
-      onQueryChange(JSON.stringify(lastQueryRequest.query))
-    }
-  }, [onQueryChange, lastQueryRequest.query])
```

Comment:
This effect was totally removed from both `QueryWrapper`s -- we already call `onQueryChange` every time `setQuery`/`executeQueryRequest` is called. Not sure why we would need this effect as well.